tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -22,4 +24,25 @@ public TesAdapterController(TesTaskService tesTaskService) {
         tesTaskService.stub();
         return new ResponseEntity<>("OK", HttpStatus.OK);
     }
+
+    @PostMapping("/v1/tasks")
+    @ResponseBody
+    ResponseEntity<TesCreateTaskResponse> submitTesTask(@RequestBody TesTask body) {
+        tesTaskService.stub();
+        return new ResponseEntity<TesCreateTaskResponse>(new TesCreateTaskResponse(), HttpStatus.NOT_IMPLEMENTED);
+    }
+
+    @GetMapping("/v1/tasks/{id}")
+    @ResponseBody
+    ResponseEntity<TesTask> getTesTask(@PathVariable("id") String id) {
```

Comment:
Value for `@PathVariable` annotation is not required, if name matches argument name.

Reply:
This method also has one optional parameter: `view` (check the [spec](https://github.com/ga4gh/task-execution-schemas/blob/master/task_execution.swagger.json)). It is better to wait until #13  will be merged as it will provide enum for this parameter.

Reply:
fixed

Reply:
fixed